app/src/main/java/eu/kanade/tachiyomi/ui/library/LibrarySort.kt

```diff
@@ -8,7 +8,8 @@ object LibrarySort {
     const val UNREAD = 3
     const val TOTAL = 4
     const val LATEST_CHAPTER = 6
-    const val DATE_ADDED = 7
+    const val CHAPTER_FETCH_DATE = 7
+    const val DATE_ADDED = 8
```

Comment:
For anyone using `Date added` this will mean their sorting filter will be changed to `Date fetched`. And this may confuse/be unwanted for some when their sorting filter suddenly changes. You probably want to keep `Date added` as `7`. `Date fetched` can still be before `Date added` in the sorting options even though it is `8`
```suggestion
    const val CHAPTER_FETCH_DATE = 8
    const val DATE_ADDED = 7
```

Or is this meant to change the sorting filtering for anyone using `Date added` to `Date fetched`?

Reply:
ah right

>Or is this meant to change the sorting filtering for anyone using Date added to Date fetched?

no